### ISSUE TITLE
p-datatable doesn't apply column width to data cells when rowGroupMode="subheader" and scrollable="true"

### ISSUE THREAD

**Issue by SpiderPork** [primefaces/primeng#2868]
**I'm submitting a ...**  (check one with "x")
```
[ X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**

http://plnkr.co/edit/fCLSwoXoSsdXRpW30DPe?p=preview

**Current behavior**
Column width is applied only to header cells, while data cells have all the same width.

**Expected behavior**
Each column maintains the width specified in the [style] for header cells and data cells.

**Minimal reproduction of the problem with instructions**
See Plunkr Case

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 4.1.3

* **PrimeNG version:** 4.0.1

* **Browser:** [all]

 
* **Language:** [TypeScript 2.3.3 - ES5]

* **Node (for AoT issues):** `node --version` = N/A

**Comment by mirko-vetrano** [comment]
Same for me, cannot find a way to fix that.

**Comment by upstreamosGH** [comment]
Yeah! Me too I'm facing with the same bug.
Seems it doesn't inject the right style in the body's cells.
I'll really appreciate this fix. ;-)

**Comment by SpiderPork** [comment]
Still present with PrimeNG 4.0.3

**Comment by cagataycivici** [comment]
Review for 4.1.1.

**Comment by SpiderPork** [comment]
It seems to be fixed in 4.1.0.
Thanks!

**Comment by cagataycivici** [comment]
Yes, unable to replicate with 4.2.2.
